package.json

```diff
@@ -25,6 +25,7 @@
     "babel-plugin-transform-decorators-legacy": "^1.3.4",
     "ember-cli-babel": "^6.0.0",
     "ember-cli-version-checker": "^2.0.0",
+    "ember-compatibility-helpers": "~0.0.2",
```

Comment:
Can we version this so we allow ^ dep? (I think it would require a 0.1.0 at least). That way we have a way to hot fix without requiring both libs to be updated.